src/legacy/status_im/profile/core.cljs

```diff
@@ -85,5 +86,8 @@
                   {:pubkey     identity
                    :ens        ens-name
                    :success-fn (fn [_]
-                                 {:dispatch [:open-modal :profile]})}]}
+                                 {:dispatch [:open-modal
```

Comment:
probably it's better to move this event `{:events [:chat.ui/show-profile]}` out of legacy codebase